### PR TITLE
storage: add windows system to GetExistingSystems

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Review by KKoukiou — COMMENTED** [review]
@vojtechtrefny do you have some better idea how to do this?

File: pyanaconda/modules/storage/devicetree/viewer.py [hunk 1]
```diff
@@ -519,15 +519,23 @@ def _get_windows_data(self):
...
+            if str(device.part_type_uuid) == EFI_PARTITION_TYPE:
+                efi_partition = device
+                continue
+
             if str(device.part_type_uuid) in WINDOWS_PARTITION_TYPES:
                 windows_data.devices.append(device.name)
 
         if len(windows_data.devices) > 0:
+            if efi_partition is not None:
+                windows_data.devices.append(efi_partition.name)
```
Comment (jkonecny12) [inline pyanaconda/modules/storage/devicetree/viewer.py:537]: If this doesn't necessarily mean that the EFI is part of the Windows then we shouldn't use `windows_data` as the variable?

Comment (KKoukiou) [reply pyanaconda/modules/storage/devicetree/viewer.py:537]: Well if there is Windows, and there is an EFI partition and you delete it windows wont boot. So you should account it in the Windows devices. 
Similarly to how Linux devices contain /boot.

Comment (KKoukiou) [reply pyanaconda/modules/storage/devicetree/viewer.py:537]: The point of this commit is that if you find an EFI partition you should not directly assume there is windows. If there is windows the EFI partition belongs to that (as well)

**Review by rvykydal — APPROVED** [review]
Looks good to me.

File: pyanaconda/modules/storage/devicetree/viewer.py (hunk 1 above)
Comment (jkonecny12) [reply pyanaconda/modules/storage/devicetree/viewer.py:537]: I see. I guess I misunderstood the code. Thanks for explanation.

**Review by jkonecny12 — APPROVED** [review]

**Pull request merged**